src/views/Editor.vue

```diff
@@ -551,12 +556,21 @@ export default {
 		'qvainData.next': {
 			handler(newNext) {
 				const newBool = !!newNext
-				if (this.$store.state.metadata.isOldVersion !== newNext) {
+				if (this.$store.state.metadata.isOldVersion !== newBool) {
```

Comment:
Looks a bit weird, is this correct?

Reply:
It's supposed to prevent commiting setMetadata with the same value that it already had, but it accidentally compared the old booleanized value to the new non-booleanized value, which will never be the same. TBH, I'm not sure why I did the check originally, it's not like the extra Vuex call there will end the world.

I''ll remove the checks.